Jenkinsfile

```
@@ -1,6 +1,6 @@
 #!/usr/bin/env groovy
 
-@Library('apm@current') _
+@Library('apm@feature/filterStage') _
```

Requires https://github.com/elastic/apm-pipeline-library/pull/1076

---

Looks to be merged now so I assume this can be updated.

---

Yes, https://github.com/elastic/apm-pipeline-library/releases/tag/v1.1.210 is now available. 

```suggestion
@Library('apm@current') _
```